Build/BUILDmom6

```diff
@@ -28,11 +28,21 @@ set -x
 # configure your build parameters
   COMPILER="intel"
   BIT="64bit"
+  COMP=""
```

Comment:
Assuming an empty `COMP` variable implies `PROD=Y`, it would be good to have a comment explaining.